features/fixtures/rack/app/app.rb

```diff
@@ -2,19 +2,41 @@
 require 'rack'
 require 'json'
 
+$clear_all_flags = false
+
 Bugsnag.configure do |config|
   config.api_key = ENV['BUGSNAG_API_KEY']
   config.endpoint = ENV['BUGSNAG_ENDPOINT']
 
   if ENV.key?('BUGSNAG_METADATA_FILTERS')
     config.meta_data_filters = JSON.parse(ENV['BUGSNAG_METADATA_FILTERS'])
   end
+
+  config.add_feature_flag(
+    'this flag is added outside of a request and so should never normally be visible',
+    'if an event was reported in the global scope, this flag would be attached to it'
+  )
```

Comment:
I'm assuming this now won't be the case once this PR has been rebased from the #754 changes? I would expect global flags to appear in all events. If that's not the case, then we probably need to discuss this.

Reply:
This branch was already based on #754. There's no such thing as a global feature flag anymore

You can make a flag always show up by adding it in an `on_error` callback, but any you add using `Bugsnag` or `Bugsnag::Configuration` are stored in a thread local so they are request/job specific